components/pkg-export-tar/src/cli.rs

```diff
@@ -0,0 +1,153 @@
+use clap::{App, Arg};
+use std::result;
+use std::str::FromStr;
+
+use url::Url;
+use common::command::package::install::InstallSource;
+
+/// The version of this library and program when built.
+pub const VERSION: &'static str = include_str!(concat!(env!("OUT_DIR"), "/VERSION"));
+
+#[derive(Clone)]
+pub struct Cli<'a, 'b>
+where
+    'a: 'b,
+{
+    pub app: App<'a, 'b>,
+}
+
+pub struct PkgIdentArgOptions {
+    pub multiple: bool,
+}
+
+impl<'a, 'b> Cli<'a, 'b> {
+    pub fn new(name: &str, about: &'a str) -> Self {
+        Cli {
+            app: clap_app!(
+            (name) =>
```

Comment:
Is it possible to use the same Clap syntax as below when initializing the `clap_app`?

Reply:
I'm not sure what you mean by that, could you expand a bit?